plugin/src/main/kotlin/tanvd/grazi/remote/downloader/LangDownloader.kt

```diff
@@ -134,14 +76,13 @@ object LangDownloader {
 
         // jars must have at least one jar with language
         if (jars.isEmpty()) {
-            // FIXME very ugly
```

Comment:
We don't plan to fix this?

Reply:
I think yes. You've convinced me that it is the ordinary way to solve the problem